kpi/serializers/v2/attachment_bulk_delete.py

```diff
@@ -0,0 +1,36 @@
+from django.utils.translation import gettext as t
+from rest_framework import serializers
+
+from kpi.exceptions import ObjectDeploymentDoesNotExist
+
+
+class AttachmentBulkDeleteSerializer(serializers.Serializer):
+    attachment_uids = serializers.ListField(
+        child=serializers.CharField(),
+        required=True,
+        help_text=t('List of attachment UIDs to delete.'),
+    )
+
+    def save(self, request, asset):
+        deployment = self._get_deployment(asset)
+        attachment_count = deployment.delete_attachments(
+            request.user, self.data['attachment_uids']
+        )
```

Comment:
Maybe we want to test to return an validation error (or access denied) if "AttachmentUidMismatchException"  is raised?